snuba/datasets/storages/transactions.py

```diff
@@ -130,9 +131,11 @@
         ArrayJoinKeyValueOptimizer("tags"),
         ArrayJoinKeyValueOptimizer("measurements"),
         ArrayJoinKeyValueOptimizer("span_op_breakdowns"),
-        ArrayJoinOptimizer(
-            "spans", ["op", "group"], ["exclusive_time"], use_bf_index=True
-        ),
+        # the bloom filter optimizer should occur before the array join optimizer
```

Comment:
For the record you could have had the bloom filter look for the conditions created by the arrayjoin. But not a big deal.